docs/pages/admin-guides/access-controls/resource-rbac.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+Per-resource RBAC enables administrators to control user access on a per-resource basis. Using a granular approach allows you to grant more precise management of user permissions in your environment.
+
+## What is per-resource RBAC?
+
+This capability is particularly useful in complex, multi-tenant environments where user groups have varying levels of access requirements. It allows for precise control over permissions, enabling restricted access to sensitive resources for certain users while granting broader access to others based on their operational needs.
+
+## Use cases
+
+### Multi-tenant environments
+
+In a multi-tenant setup, per-resource RBAC can ensure that each client or team has access only to their specific resources, preventing any accidental or intentional cross-tenant access.
+
+### Compliance requirements
+For industries with strict regulatory requirements (e.g., finance, healthcare), per-resource RBAC helps in implementing and demonstrating compliance with access control policies.
+
+### Team management
+For new team members or roles, per-resource RBAC facilitates a gradual increase in access privileges as users become more familiar with the systems and demonstrate need and competence.
+When working with external contractors, per-resource RBAC allows granting temporary, limited access to specific resources without exposing the entire infrastructure.
+
+### DevOps workflows
+In CI/CD pipelines, different stages may require different levels of access; per-resource RBAC can be used to grant just-in-time access for specific tasks in the pipeline.
+
+Let's look at some examples of how per-resource RBAC can be implemented:
+
```

Comment:
Would it make sense to merge this information into the role reference or the RBAC getting started guide? Aside from the description of benefits and use cases, it seems like the examples below could work in one of those guides.

Reply:
I didn't want to make the getting started page too long, but the content does work well together. Removed some of the redundancies and combined them, lmk what you think